assets/sass/components/analytics-4/audience-segmentation/_googlesitekit-audience-segmentation-tile.scss

```diff
@@ -17,6 +17,11 @@
  */
 
 .googlesitekit-plugin {
+	.googlesitekit-widget.googlesitekit-audience-segmentation-tile--zero-data {
+		border-bottom-left-radius: 0;
+		border-bottom-right-radius: 0;
+	}
+
```

Comment:
As per my other comment, I don't think we'll want square corners for this version of the widget and we should be able to remove this.

```suggestion
```

Reply:
Removed.